### ISSUE TITLE
Response view not showing

### ISSUE THREAD

**Issue by Razzeee** [jcoc611/vsconnect#6]
Hey there, I've tried to use this a few times, but I never managed to get a response window to show up after sending a query. I'm only getting the next request window.

**Comment by jcoc611** [comment]
Thanks for reporting the issue! If you have time, could you let me know what your setup is like? (OS and vscode version?). Also, does it not work for every website (e.g. even https://google.com doesn't work?).

**Comment by Razzeee** [comment]
That was from windows using the latest vscode version 1.50.1 it seemed to not work on any site. I think I tried google, will let you know tomorrow.

I also tried to install this on my linux setup right now, but I can't even find the extension via vscode (1.50.1) is that supposed to be like that? If I go to https://marketplace.visualstudio.com/items?itemName=jcoc611.vsconnect download the vsix and install it, it installs fine and even works fine with google.com

**Comment by Razzeee** [comment]
Tried on the windows client again and it seems like google works (now) it just fails to show anything for some the local network sites. I can get it working for my fritz.box url, but my NAS (synology) doesn't seem to respond. I also fail to use it on work related stuff, there's a vpn involved, not sure if that matters.

**Comment by jcoc611** [comment]
I have made some changes on `v0.2.10` that should improve the overall experience. I added error messages on HTTP connection errors so it is easier to tell what is going wrong, and added some better parsing of URLs that was causing some issues.

I tried accessing sites over VPN (on Windows), and had no problems after the changes. Let me know if you still run into issues!

Thanks again, and sorry for the long delay!
